<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QMainWindow" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EScreenShot-Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBarTools">
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRect"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionLine"/>
   <addaction name="actionArrow"/>
   <addaction name="actionText"/>
   <addaction name="actionDraw"/>
  </widget>
  <widget class="QToolBar" name="toolBarColor">
   <property name="windowTitle">
    <string>DrawColor</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionColorRed"/>
   <addaction name="actionColorGreen"/>
   <addaction name="actionColorBlue"/>
   <addaction name="actionColorGray"/>
   <addaction name="actionColorWhite"/>
   <addaction name="actionColorBlack"/>
   <addaction name="actionColorCustom"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
  </widget>
  <action name="actionRect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/rc/images/icon_rect.png</normaloff>:/rc/images/icon_rect.png</iconset>
   </property>
   <property name="text">
    <string>Rect</string>
   </property>
   <property name="toolTip">
    <string>Rect</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/rc/images/icon_ellipse.png</normaloff>:/rc/images/icon_ellipse.png</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/rc/images/icon_line.png</normaloff>:/rc/images/icon_line.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionArrow">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/rc/images/icon_arrow.png</normaloff>:/rc/images/icon_arrow.png</iconset>
   </property>
   <property name="text">
    <string>Arrow</string>
   </property>
  </action>
  <action name="actionDraw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/rc/images/icon_draw.png</normaloff>:/rc/images/icon_draw.png</iconset>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </action>
  <action name="actionText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/rc/images/icon_text.png</normaloff>:/rc/images/icon_text.png</iconset>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
   <property name="toolTip">
    <string>Text</string>
   </property>
  </action>
  <action name="actionColorBlack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ColorBlack</string>
   </property>
   <property name="toolTip">
    <string>Black</string>
   </property>
  </action>
  <action name="actionColorWhite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ColorWhite</string>
   </property>
   <property name="toolTip">
    <string>White</string>
   </property>
  </action>
  <action name="actionColorGray">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ColorGray</string>
   </property>
   <property name="toolTip">
    <string>Gray</string>
   </property>
  </action>
  <action name="actionColorRed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ColorRed</string>
   </property>
   <property name="toolTip">
    <string>Red</string>
   </property>
  </action>
  <action name="actionColorGreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ColorGreen</string>
   </property>
   <property name="toolTip">
    <string>Green</string>
   </property>
  </action>
  <action name="actionColorBlue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ColorBlue</string>
   </property>
   <property name="toolTip">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionColorCustom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/rc/images/icon_color_custom.png</normaloff>:/rc/images/icon_color_custom.png</iconset>
   </property>
   <property name="text">
    <string>ColorCustom</string>
   </property>
   <property name="toolTip">
    <string>Custom</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>Editor</receiver>
   <slot>slotActionSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotActionSave()</slot>
 </slots>
</ui>
